agent/src/beerocks/monitor/monitor_stats.cpp

```diff
@@ -51,7 +51,7 @@ void monitor_stats::add_request(uint16_t id, uint8_t sync)
         }
         next_poll_id = mon_db->get_poll_id() + 1;
     }
-    requests_list.push_back(id);
+    requests_list.push_back(sMeasurementsRequest(id, sta_mac));
```

Comment:
```suggestion
    requests_list.push_back({ id, sta_mac });
```
With that, the constructor is not needed.